cli/cmd/upgrade/upgrade.go

```diff
@@ -232,7 +232,10 @@ func (c *upgradeSpec) UpgradePathBasedOnVersion() error {
 		return fmt.Errorf("failed to parse 0.2.2 version %w", err)
 	}
 
-	if nVersion >= version0_4_0 {
+	// kube-prometheus is introduced on tobs >= 0.4.0 release
+	// so create CRDs if version >= 0.4.0 and only create CRDs
+	// if version change is noticed in upgrades...
+	if nVersion >= version0_4_0 && nVersion != dVersion {
```

Comment:
hmm so CRDs will be re-created every time? Then the crd urls on line 470 that have "v0.47.0" encoded in the URL are probably not appropriate. Thats probably a bug we should fix in this pr

Reply:
Oh nevermind I see how that's supposed to work